### ISSUE TITLE
accounts-password@1.5.2: Accounts.addEmail fails if user has no emails

### ISSUE THREAD

**Issue by Nidroide** [meteor/meteor#10815]
Since underscore was removed Accounts.addEmail fails on users without emails:
```
const didUpdateOwnEmail = user.emails.reduce(
    (prev, email) => {
...
```
due calling reduce on undefined.
Regards

**Comment by afrokick** [comment]
Fixed in https://github.com/meteor/meteor/pull/10816

Thanks, @Nidroide

**Comment by filipenevola** [comment]
Published accounts-password@1.5.3.
